artifactory_cleanup/rules/keep.py

```diff
@@ -143,7 +143,7 @@ class KeepLatestVersionNFilesInFolder(Rule):
     The definition of the version is using regexp. By default ``r'[^ \d][[\._]]()((\d+\.)+\d+)')``
     """
 
-    def __init__(self, count, custom_regexp=r"[^\d][\._]((\d+\.)+\d+)"):
+    def __init__(self, count, custom_regexp=r"([\d]+\.[\d]+\.[\d]+)"):
```

Comment:
Could you fix the readme please for that case please? https://github.com/devopshq/artifactory-cleanup/blob/master/README.md?plain=1#L253

Let's say `By default it parses semver https://semver.org/` instead of actual regexp

Reply:
Added to the README. I did also include the specific regex it is using to parse semver for completeness.